yarn.lock

```diff
@@ -16801,6 +16801,11 @@ libnpmpublish@^4.0.0:
     semver "^7.1.3"
     ssri "^8.0.1"
 
+libphonenumber-js@1.9.47:
```

Comment:
This is a fairly large dependency (7.11Mb unpacked according to [npm](https://www.npmjs.com/package/libphonenumber-js), can it be deduped with `^1.7.31`?

Reply:
I had to pin this to latest version which builds. There has been an update to this package and they had had a breaking change which fails compilation